From 096c31527b960585a7e0a459b911363248d67655 Mon Sep 17 00:00:00 2001
From: Maxime Hadjinlian <maxime.hadjinlian@gmail.com>
Date: Sun, 12 Oct 2014 16:22:31 +0200
Subject: [PATCH] pkgconfig: Do not specify full path

The fullpath ends up at /usr/... which points to the host machine.

Signed-off-by: Maxime Hadjinlian <maxime.hadjinlian@gmail.com>
---
 js/src/js.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/src/js.pc.in b/js/src/js.pc.in
index 4debb8b..9b06d92 100644
--- a/js/src/js.pc.in
+++ b/js/src/js.pc.in
@@ -8,4 +8,4 @@ Description: The Mozilla library for JavaScript
 Version: @MOZILLA_VERSION@
 Requires.private: @NSPR_PKGCONF_CHECK@
 Libs: -L${libdir} -l@LIBRARY_NAME@
-Cflags: -include ${includedir}/@LIBRARY_NAME@/js/RequiredDefines.h -I${includedir}/@LIBRARY_NAME@
+Cflags: -include @LIBRARY_NAME@/js/RequiredDefines.h -I${includedir}/@LIBRARY_NAME@
-- 
2.1.1

